test/scales/scaleTests.ts

```diff
@@ -140,6 +142,23 @@ describe("Scales", () => {
         "Unloading the CSS should cause color scales fallback to default colors");
     });
 
+     it("caches CSS specified colors unless the cache is invalidated", () => {
+      var style = d3.select("body").append("style");
+      style.html(".plottable-colors-0 {background-color: #ff0000 !important; }");
+      Plottable.Scales.Color.invalidateColorCache();
```

Comment:
The rest of the tests already written use the full name with Plottable, so I stuck with that convention.